### DISCUSSION TITLE
Welcome to Crypto-Triangular-Arbitrage-Bot Discussions!

### DISCUSSION THREAD

**Discussion by MfaXyz** [MfaXyz/Crypto-Triangular-Arbitrage-Bot#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rashimak** [comment]
Sir,
Can i use the triangular arbitrage bot for android phone?

**Reply by MfaXyz** [reply]
Yes.

**Comment by rashimak** [comment]
How to install?
